pallets/msa/src/mock.rs

```diff
@@ -166,24 +166,33 @@ pub fn create_and_sign_add_provider_payload(
 }
 
 /// Creates a provider and delegator MSA and sets the delegation relationship.
+// create and set up delegations for a delegator and provider, but for convenience only
 /// # Returns
 /// * (u8, Public) - Returns a provider_msa_id and a delegator account.
-pub fn test_create_delegator_msa_with_provider() -> (u64, Public) {
-	let (key_pair, _) = sr25519::Pair::generate();
-
-	let provider_account = key_pair.public();
-
-	let (delegator_key_pair, _) = sr25519::Pair::generate();
+pub fn create_provider_msa_and_delegator() -> (u64, Public) {
+	let (provider_msa_id, _, _, delegator_account) = create_provider_delegator_msas();
+	(provider_msa_id, delegator_account)
+}
 
-	let delegator_account = delegator_key_pair.public();
+// create and set up delegations for a delegator and provider, but for convenience only
+// return delegator msa and provider account for testing delegator-submitted extrinsics
+/// # Returns
+/// * (u8, Public) - Returns a delegator_msa_id and a provider_account.
+pub fn create_delegator_msa_and_provider() -> (u64, Public) {
```

Comment:
This was renamed to make more sense, especially because I wanted to use its new counterpart.